screens/DataVerificationScreen.tsx

```diff
@@ -46,27 +43,33 @@ const DataVerificationScreen = (): JSX.Element => {
     }
   };
 
+  const getDraftStepData = (id: number) => {
```

Comment:
You're using this to get data to prop drill into DataVerifItem, but it's not being used in that component. I think it makes more sense to remove it from this component, since it's not needed.